packages/atomic/src/components/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -0,0 +1,97 @@
+import {Component, Element, State, h} from '@stencil/core';
+import {buildInstantResults, InstantResults, Result} from '@coveo/headless';
+
+import {
+  dispatchSearchBoxSuggestionsEvent,
+  SearchBoxSuggestionItem,
+  SearchBoxSuggestions,
+  SearchBoxSuggestionsBindings,
+} from '../suggestions-common';
+
+/**
+ * The `atomic-search-box-instant-results` component can be added as a child of an `atomic-search-box` component, allowing for the configuration of instant results behavior.
+ * @internal
```

Comment:
so basically that means that we won't expose this component in the public documentation, right?

It also explains, why this components' parts are not described in this file, but in the [atomic-search-box.tsx](https://github.com/coveo/ui-kit/pull/1999/files#diff-46e3c6b20a52a198d73e328db1b88c9a1b4b44ba84c5e7556011e630fc8f4638)  instead.

If I understand all this correctly, the new component is sort of "service" / "under-the-hood" component.

It all makes sense to me, I just wanted to double-check if it's what we want for our users 😊🙏

@liviarett

Reply:
@npushkarskii it will be exposed actually, but it's still under construction 🚧

Reply:
aha, got it, thx!

 If it's going to be exposed, don't we want to specify describe parts of that component in `atomic-search-box-instant-results.tsx`? Pretty much like we did for `atomic-search-box-recent-queries`.

@liviarett 

![image](https://user-images.githubusercontent.com/92988368/170774263-6043468a-3009-4ac5-af9f-1aec71436b7e.png)

Reply:
ah, so we did this because the recent queries has a header that we wanted to be able to target. but this one doesn't have anything special, at least for now! no parts available there, so it's ok